README.md

```diff
@@ -133,3 +144,10 @@ Options:
                                 into which uploads are deposited.
   --help                        Show this message and exit.
 ```
+
+**Example method call**: 
+
+```bash
+pipenv run dsaps --url $DSPACE_URL -e $DSPACE_EMAIL -p $DSPACE_PASSWORD additems -m <metadata-csv> -f config/<field-mapping>.json -d <s3-bucket-name> -t <file-type> -c <collection-handle>
+```
+
```

Comment:
I think it would be useful to have the full command that is usually run so it's clear that you can combine `newcollection` and `additems` into a single command (and noting that you can't use `-c <collection-handle>` if `additems` is run with `newcollection`):

```
pipenv run dsaps --url $DSPACE_URL -e $DSPACE_EMAIL -p $DSPACE_PASSWORD newcollection -c <community-handle> -n <collection-name> additems -m <metadata-csv> -f config/<field-mapping>.json -d <s3-bucket-name> -t <file-type>
```